tests/pytests/test_reporter.py

```diff
@@ -38,7 +38,21 @@ def get_vals(self, idx):
         return self.vals[:, idx]
 
 
-def test_simple_report():
+class FakeBloodflowManager:
```

Comment:
Would be interesting to use the mock Python package, but for this one-time usage, that's fine.

Reply:
I followed the same idea as for the other report